src/EnergyPlus/MixedAir.hh

```diff
@@ -311,9 +315,8 @@ namespace MixedAir {
         Real64 Evz = 1.0;      // zone ventilation efficiency
         Real64 SysDesOA = 0.0; // System design OA
 
-        void CalcMechVentController(EnergyPlusData &state,
-                                    Real64 &SysSA,             // System supply air mass flow rate [kg/s]
-                                    Real64 &MechVentOAMassFlow // outside air mass flow rate calculated by mechanical ventilation object [kg/s]
+        Real64 CalcMechVentController(EnergyPlusData &state,
+                                      Real64 &SysSA // System supply air mass flow rate [kg/s]
```

Comment:
I don't think this needs to be passed by reference.

Reply:
I thought the same at first, but this function does have places that change `SysSA` (not that I understand why).

Reply:
Ok, but how does that flow back up into the rest of the system?

Reply:
> I thought the same at first, but this function does have places that change `SysSA` (not that I understand why).

Oh, I didn't see that. Sorry.

Reply:
If you are not using the modified value then yes.